### PR TITLE
added rockspec

### PR DESCRIPTION
**Pull request by SatyendraBanjare**
Adding rockspec for installing via luarocks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,18 @@ Though the project is still under development, you can try, install and test the
...
-First, install [libgit2](https://github.com/libgit2/libgit2), according to the instructions on their documentation.
+- First, install [libgit2](https://github.com/libgit2/libgit2), according to the instructions on their documentation.
+
+- Since this project depends on LuaC-Api , it is required to install `liblua5.1-dev`, `liblua5.2-dev` , `liblua5.3-dev` 
+as :
+
+```
+sudo apt-get install liblua5.1-dev
+sudo apt-get install liblua5.2-dev
+sudo apt-get install liblua5.3-dev
```
Comment (Etiene) [inline README.md:21]: Why having all of the lua versions?

Comment (Etiene) [reply README.md:21]: Just the version the user wants should be enough, no?

Comment (SatyendraBanjare) [reply README.md:21]: @Etiene , in my rockspec file I have included all the three i.e lua5.1, 5.2 , 5.3. so liblua-dev for all these should be present for error free installation. alternatively user will have to remove version number for which he does not want installation.

File: luagit2-0.0-1.rockspec [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+dependencies = {
+   "lua >= 5.1, < 5.4",
+}
+
+build = {
+   type = "builtin",
+   modules = {
+      luagit2 = {
+         sources = {"src/main_wrapper.c" },
+         libraries = {"git2" , "lua5.1" , "lua5.2" , "lua5.3" },
```
Comment (SatyendraBanjare) [inline luagit2-0.0-1.rockspec:20]: @Etiene  here it  is :smile:

**Comment by SatyendraBanjare** [comment]
Thank you so much for reviewing :smiley:

**Pull request merged**